.github/workflows/process_test.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"        
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          path: "main"
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies
+        shell: bash
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel
+          cd $GITHUB_WORKSPACE/main
+          pip install .  
+      - name: Clone AWS Level 0 data repo for testing
+        env:
+          GITLAB_TOKEN : ${{ secrets.GITLAB_TOKEN }}
+        run: | 
+          cd $GITHUB_WORKSPACE
+          git clone --depth 1 https://oauth2:${{ env.GITLAB_TOKEN }}@geusgitlab.geus.dk/glaciology-and-climate/promice/aws-l0.git
+      - name: Run data processing
+        env:
+          TEST_STATION: KPC_U CEN2 JAR
```

Comment:
If it is not a huge time constraint, maybe we run this for two of each station type? Just seems like a bit more redundancy here would be good?